core/src/test/scala/org/typelevel/paiges/PaigesScalacheckTest.scala

```diff
@@ -162,83 +162,12 @@ class PaigesScalacheckTest extends FunSuite {
     }
   }
 
-  test("c.flatten + d.flatten == (c + d).flatten") {
```

Comment:
why do we need to remove all these tests? We still have a private flatten function, do these laws no longer pass?

Reply:
Some are still true.  I just didn't think we wanted to test implementation details here.  I'll return those.